### PR TITLE
Update Readmes

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
Update top-level readme and dockers' readme, and add missing readme files in man/ and doc/.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes issue linked in Description above)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance)
- [ ] Other

### Checklist

- [ ] Code compiles without errors
- [ ] All tests pass locally with my changes (see TESTING section in CONTRIBUTING file)
- [ ] Created tests which will fail without the change (if possible)
- [x] Extended the README/documentation (if necessary)
- [ ] All newly added files have proprietary license (if necessary)
- [ ] All newly added files are referenced in MANIFEST files (if necessary)

- - -
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/863)

### CODE REVIEW

File: utils/docker/README.md [hunk 1]
```diff
@@ -53,14 +56,37 @@ memkind to not release memory to OS in anticipation of memory reuse soon. For PM
...
 ```
 
 **Note:**
 
-Others class group must have write permission to MEMKIND_HOST_WORKDIR to use the run_local script.
+Others class group must have write permission to *MEMKIND_HOST_WORKDIR* to use the *run_local.sh* script.
+
+### Building docker images manually
+
+Images built out of Dockerfiles ("recipes") found in this directory may by used with `docker` or `podman` as
```
Comment (PatKamin) [inline utils/docker/README.md:73]: may *be* used

File: doc/README [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# doc files
+
+This directory contains documentation files for memkind interfaces in
+a human readable format (using Markdown syntax).
+
+To see a generated manpages (out of these docs), please see [man directory](../man/).
```
Comment (PatKamin) [inline doc/README:6]: To see generated manpages

**Review by PatKamin — APPROVED** [review]

**Review by lukaszstolarczuk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/863)* status: 0 of 5 files reviewed, 2 unresolved discussions (waiting on @PatKamin)

___
*[`doc/README` line 6 at r3](https://reviewable.io/reviews/memkind/memkind/863#-NJytBAa9qm4CCtF5O3d-r3-6:-NKrBsNY4qhqBbAuIDoI:buzbreo) ([raw file](https://github.com/memkind/memkind/blob/2d93cc849fecff696e8c9e2199c66558edb37aee/doc/README#L6)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

To see generated manpages
</blockquote></details>

Thanks & done.
___
*[`utils/docker/README.md` line 73 at r2](https://reviewable.io/reviews/memkind/memkind/863#-NJytBAa9qm4CCtF5O3b-r2-73:-NKrBsduDgmLzPu3NwnL:b-896fix) ([raw file](https://github.com/memkind/memkind/blob/ea3d3bc644a7cef86fef30ee22da8f4b7defd059/utils/docker/README.md#L73)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

may *be* used
</blockquote></details>

Done.

**Review by PatKamin — APPROVED** [review]
> Reviewed 1 of 1 files at r1, 3 of 4 files at r4, 3 of 3 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/863)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @lukaszstolarczuk)

**Review by szadam — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/863)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @lukaszstolarczuk)

**Pull request merged**